go/obscuronode/enclave/enclave.go

```diff
@@ -544,6 +615,29 @@ func (e *enclaveImpl) ShareSecret(att *obscurocommon.AttestationReport) (obscuro
 	return e.encryptSecret(att.PubKey, secret)
 }
 
+func (e *enclaveImpl) AddViewingKey(viewingKeyBytes []byte, signature []byte) error {
```

Comment:
Most of this method is copied over from the old (now-deleted) Obscuro facade. See https://github.com/obscuronet/go-obscuro/blob/58751aa7bde8397238c01b52a94430a25114e6d1/tools/walletextension/obscuro_facade.go#L54.